calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/camera_calibrator.py

```diff
@@ -606,10 +535,28 @@ def draw_evaluation_heatmap_callback(value):
             self.image_view.set_draw_evaluation_heatmap(value == Qt.Checked)
             self.should_process_image.emit()
 
+        def draw_linearity_heatmap_callback(value):
+            self.image_view.set_draw_linearity_heatmap(value == Qt.Checked)
+            self.should_process_image.emit()
+
+        def on_restart_linearity_heatmap_clicked():
+            print("restart_lin_heatmap", flush=True)
```

Comment:
delete or use logging

Reply:
thanks, i will fix it

Reply:
f2ad5cce4cffc72284d0b575020c40851475872e